system-contracts/last_compilation_bootloader.timestamp

```diff
@@ -1 +1 @@
-1718185700858
```

Comment:
just in case, could you please remove changes for `timestamp`? We don't use hardhat anymore so this thing is not needed. What's more, we probably shouldn't commit it even as it was intended for local compiler caching

Reply:
That makes sense. Will make this change in the next PR.